routing/ROUTING_V1_HTTP.md

```diff
@@ -90,6 +90,18 @@ Response limit: 100 providers
 
 Each object in the `Providers` list is a *read provider record*.
 
+#### List of Routers
+
+A list of routers that should be consulted for responses. The specification imposes no constraints on the order by which the results are returned. Implementers are free to return results as they are found.
+
+If none are provided the backend may decide which ones to use. For a list of known routers see below.
+
+To select all routers use `?routers=*`. To determine the list of available routers see the `OPTIONS` API below
+
+### `OPTIONS /routing/v1/providers`
+
+If supported the server should include a `SUPPORTED_ROUTERS` header key, with the value as the comma separated list of the routing systems supported.
```

Comment:
nit: if this is a HTTP header, let's use the [canonical convention](https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/net/http/header.go;l=240):
```suggestion
If supported the server should include a `Supported-Routers` HTTP header, with the value as the comma separated list of the routing systems supported.
```

Reply:
Prefix the headers with `X-Ipfs-` or something similar?

Reply:
How do people feel about 
```suggestion
If supported the server should include an `Ipfs-Supported-Routers` header key, with the value as the comma separated list of the routing systems supported.
```